### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by Priceless-P** [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi#13] [Outreachy 2022] [answered]
Hi everyone :clap:,

My name is Prisca Chidimma Maduka. I am one of the Outreachy Applicants. I’m looking forward to learning and, making meaningful contributions. I’m so excited to be here. :star_struck:

**Comment by BatoolMM** [comment, marked as answer]
Welcome @Priceless-P to the community!

We have two projects for Outreachy, you can find them here:
1- [Improve the bilinguality and accessibility of the website](https://github.com/Open-Science-Community-Saudi-Arabia/OSCSA_Website/issues/12)
2- Improve Localisation for [Open Educational Resources](https://github.com/Open-Science-Community-Saudi-Arabia/localisation/issues/13).

Feel free to comment under the issue, if you have any questions!

**Reply by Priceless-P** [reply]
Thank you very much @BatoolMM 

I will be contributing to the first project:
1- [Improve the bilinguality and accessibility of OSCSA website](https://github.com/Open-Science-Community-Saudi-Arabia/OSCSA_Website/issues/12)
